core/domain/value_generators_domain_test.py

```diff
@@ -48,6 +48,31 @@ def test_generate_value_of_base_value_generator_raises_error(self) -> None:
                 'generate_value() method has not yet been implemented')):
             base_generator.generate_value()
 
+    def test_class_name_of_value_generator(self) -> None:
+        class_object = value_generators_domain.BaseValueGenerator()
+        self.assertEqual('BaseValueGenerator', class_object.__class__.__name__)
```

Comment:
I don't see a point of this test.

Reply:
I wrote it solely as it stated it didn't have coverage. It can be dropped if you feel it isn't needed.